beancount_import/source/ofx.py

```diff
@@ -1336,6 +1328,22 @@ def _process_journal_entries(self):
                     InvalidSourceReference(excess_number, transactions))
 
 
+def _findOfxId(account, account_to_ofx_id):
```

Comment:
Can you change to `snake_case` for style consistency?

Reply:
Oops, had Go on my mind. Done. I've also added a second commit with a regression test that fails without the fix.